### PR TITLE
Query Browser: Add empty state and suggested query

### PR DESCRIPTION
**Pull request by kyoto**
Add an empty state for the case where no query has been entered and also
a suggested example query.

screenshot

### CODE REVIEW

**Comment by spadgett** [comment]
A doc link with some example queries might be helpful here. I don't know if there's anything like that today or something new we'd have to write.

It feels like the buttons should be flush with the edges of the sections above and below.

cc @cshinn @bmcelvee

File: frontend/public/components/monitoring/metrics.tsx [hunk 1]
```diff
@@ -438,12 +453,21 @@ export const QueryBrowserPage = withFallback(() => {
...
-            disabledSeries={_.map(queries, 'disabledSeries')}
-            onDataUpdate={onDataUpdate}
-            queries={_.map(queries, 'query')}
-          />
+          {queryStrings.join('') === ''
+            ? <div className="query-browser__wrapper graph-empty-state">
+              <EmptyState variant={EmptyStateVariant.full}>
+                <EmptyStateIcon size="sm" icon={ChartLineIcon} />
+                <Title size="sm">No Query Entered</Title>
+                <EmptyStateBody>Enter a query in the box below to explore metrics for this cluster</EmptyStateBody>
```
Comment (spadgett) [inline frontend/public/components/monitoring/metrics.tsx]: nit: we usually include a period if it's a complete sentence

```suggestion
                <EmptyStateBody>Enter a query in the box below to explore metrics for this cluster.</EmptyStateBody>
```

File: frontend/public/components/monitoring/metrics.tsx [hunk 2]
```diff
@@ -438,12 +453,21 @@ export const QueryBrowserPage = withFallback(() => {
...
+                <Button onClick={insertExampleQuery} variant="primary">Insert Example Query</Button>
```
Comment (spadgett) [inline frontend/public/components/monitoring/metrics.tsx]: The different button styles are a little odd, but @sg00dwin is working on converting all buttons to PF4.

**Comment by kyoto** [comment]
Button alignment addressed by https://github.com/openshift/console/pull/2094

File: frontend/public/components/monitoring/metrics.tsx (hunk 1 above)
Comment (kyoto) [reply frontend/public/components/monitoring/metrics.tsx]: Addressed by https://github.com/openshift/console/pull/2096

**Pull request merged**